test/db/formats/elf/obj

```diff
@@ -5,6 +5,8 @@ CMDS=ie~addr
 EXPECT=<<EOF
 vaddr=0x00000000 paddr=0x00000040 baddr=0x00000000 laddr=0x00000000 haddr=0x00000018 type=program
 EOF
+EXPECT_ERR=<<EOF
```

Comment:
is this needed if its empty?

Reply:
Yes, It tests that stderr is empty afaik. If you don't put it stderr is ignored, isn't it?